WordPress/WordPressUITests/Tests/EditorAztecTests.swift

```diff
@@ -18,6 +19,7 @@ class EditorAztecTests: XCTestCase {
             EditorFlow.returnToMainEditorScreen()
             editorScreen.closeEditor()
         }
+        LoginFlow.logoutIfNeeded()
```

Comment:
Was there a problem with the tests that this solves? We originally decided not to log out between each editor test because it shouldn't be needed and the extra login/logout steps slow down the tests.

Reply:
I noticed, at least locally, that some tests were failing if the user was already logged in.

Reply:
Hm, that's odd! I'm ok with leaving this change, then, and I opened https://github.com/wordpress-mobile/WordPress-iOS/issues/13547 to follow up on this.